formats/detailed_guide/publisher/details.json

```diff
@@ -0,0 +1,73 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "first_public_at"
+  ],
+  "properties": {
+    "body": {
+      "type": "string"
+    },
+    "first_public_at": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "image": {
+      "$ref": "#/definitions/image"
+    },
+    "scheduled_publication": {
+      "type": "string"
+    },
+    "excluded_nations": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": ["england", "scotland", "wales", "northern_ireland"]
+      }
+    },
+    "change_note": {
```

Comment:
Case Studies have it, but I agree we don't need it.

If we do decide to remove it, worth noting: `first_public_at` is also called as such to maintain consistency with Case Studies, but I feel it should be `first_published_at`.